doc/conf.py

```diff
@@ -395,7 +395,8 @@ def setup(app):
     # each code block
     'capture_repr': ('_repr_html_', '__repr__'),
     'matplotlib_animations': True,
-    'image_srcset': ["2x"]
+    'image_srcset': ["2x"],
+    'nested_sections': False,
```

Comment:
Why disable this for our own doc builds? It seems like we'd want this cool new feature, no?

Reply:
I agree it would be nice to use it in our doc, but nested sections don't play well with gallery items present in the root folder. The main reason is that 2 there will be 2 toctrees in the main `index.rst` (one for these gallery items and one for the gallery's sections) and that they don't interact well.

I think we have 3 ways of handling this:
- disable `nested_sections` (what I chose here)
- create a new folder in the root directory and move root gallery items there (what I chose for `nilearn`) even if this breaks html links (because file paths were changed)
- have users split their root `readme.txt` file into 2 text files: one describing the gallery, and one describing the "root section" (ie the section root items belong to). They could be named `readme_gallery.txt` and `readme.txt` respectively to be consistent across sections. This way, I could have a single toctree in the main `index.rst`, and that would most likely solve our problem!